internal/executor/testdata/cache/.cirrus.yml

```
@@ -0,0 +1,17 @@
+container:
+  image: debian:latest
+
+fill_cache_task:
+  test_cache:
+    folder: cache
+    populate_script:
+      - mkdir cache
+      - touch cache/from-populate-script
```

We can do better! Let's store a 100Mb file inside `cache`

```suggestion
      - cd cache
      - dd if=/dev/urandom of=from-populate-script bs=10M count=10
```